airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/errorreporter/JobErrorReporter.java

```diff
@@ -268,23 +273,26 @@ private void reportJobFailureReason(@Nullable final StandardWorkspace workspace,
                                       final FailureReason failureReason,
                                       final String dockerImage,
                                       final Map<String, String> metadata) {
-    final Map<String, String> commonMetadata = new HashMap<>(Map.ofEntries(
-        Map.entry(AIRBYTE_VERSION_META_KEY, airbyteVersion),
-        Map.entry(DEPLOYMENT_MODE_META_KEY, deploymentMode.name())));
-
-    if (workspace != null) {
-      commonMetadata.putAll(getWorkspaceMetadata(workspace.getWorkspaceId()));
-    }
+    // Failure types associated with a config-error or a manual-cancellation should NOT be reported.
+    if (!UNSUPPORTED_FAILURETYPES.contains(failureReason.getFailureType())) {
```

Comment:
Is this condition backwards?

Reply:
Oh nvm, the double negative threw me off 😄 (not not supported)

Reply:
Ha! Changed it back for readability..